cwms-data-api/src/main/java/cwms/cda/api/CountyController.java

```diff
@@ -100,7 +93,7 @@ public void getAll(@NotNull Context ctx) {
             CountyDao dao = new CountyDao(dsl);
             List<County> counties = dao.getCounties();
             String formatHeader = ctx.header(Header.ACCEPT);
-            ContentType contentType = Formats.parseHeader(formatHeader, CONTENT_TYPE_ALIAS_MAP);
+            ContentType contentType = Formats.parseHeader(formatHeader, ContentTypeAliasMap.forDtoClass(County.class));
```

Comment:
Can Formats do the ContentTypeAliasMap.forDtoClass call internally?  Not saying you have to but I'm curious if there are reasons it should be passed in - if the Formats method was parseHeader(formatHeader, klass.class)  then ContentTypeAliasMap would be mostly a private implementation detail -right?

Reply:
Hah, check the latest changes.  I was already ahead of you on that.  ;-)